plugins/out_es/es_conf.c

```diff
@@ -227,13 +228,20 @@ struct flb_elasticsearch *flb_es_conf_create(struct flb_output_instance *ins,
         }
     }
 
+    /* Elasticsearch: Path */
+    path = flb_output_get_property("path", ins);
+    if (!path)
```

Comment:
Minor style change is required, it should be:

```
    if (!path) {
```

Reply:
Changed. PTAL